.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Please follow the [Pull Request Process](https://backstage.iad.w10e.com/docs/default/component/eng-handbook/practices-processes/pull-request-reviews/#pull-request-process) while creating your Pull Request. You can use the below template to help you structure your thoughts.
+
+### Why
+
+[Replace with a short description of why is this change required. The “why” tells us what business or engineering goal this change achieves. The “why” is a chance to explain both the engineering goal and some business objective that is satisfied or moved along. This is also the record of the company's decision-making process, beneficial when looking back.]
```

Comment:
I think what we may want to do here is put the text in the sections in comment blocks instead, ie surround by `<!-- -->`